testsuite/shared/src/main/java/org/jboss/as/test/shared/util/LoggingUtil.java

```diff
@@ -71,26 +72,35 @@ public static Path getInServerLogPath(String logFile) {
         return Paths.get(System.getProperty("jboss.server.log.dir")).resolve(logFile);
     }
 
-    public static boolean hasLogMessage(String logFileName, String logMessage) throws Exception {
+
+    @SafeVarargs
+    public static boolean hasLogMessage(String logFileName, String logMessage, Predicate<String>... filters) throws Exception {
 
         Path logPath = LoggingUtil.getInServerLogPath(logFileName);
         return isMessageInLogFile(logPath, logMessage);
     }
 
-    public static boolean hasLogMessage(ManagementClient managementClient, String handlerName, String logMessage) throws Exception {
+    @SafeVarargs
+    public static boolean hasLogMessage(ManagementClient managementClient, String handlerName, String logMessage, Predicate<String>... filters) throws Exception {
```

Comment:
`filters` method argument should be passed to `isMessageInLogFile` as an argument

Reply:
Fixed